bazel/repositories.bzl

```diff
@@ -67,8 +67,7 @@ def _repository_impl(name, **kwargs):
 
 def envoy_gloo_dependencies():
     _repository_impl("envoy", patches=[
-        "@envoy_gloo//bazel/foreign_cc:0001-otel-exporter-status-code-fix.patch",
-        "@envoy_gloo//bazel/foreign_cc:0002-ratelimit-filter-state-hits-addend.patch",
+        "@envoy_gloo//bazel/foreign_cc:0001-ratelimit-filter-state-hits-addend.patch",
```

Comment:
what's the plan for the hits-addend patch? `0001-ratelimit-filter-state-hits-addend.patch` is currently an empty file

Reply:
Ugh thanks. Yeah thats what reviews are for. Updating